src/components/Articles/TagsComponent.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Collection } from 'react-materialize';
+import Materialize from 'materialize-css';
+import PropTypes from 'prop-types';
+import '../../assets/styles/TagsComponent.scss';
+
+
+class TagsComponent extends Component {
+  componentDidMount() {
+    const { onTagsChange, tags } = this.props;
+    const data = (tags) || [];
+    const tagsElement = document.getElementById('tags');
+    this.instance = Materialize.Chips.init(
+      tagsElement, {
+        placeholder: 'Enter a tag',
+        secondaryPlaceholder: '+Tag',
+        data,
+        onChipAdd: () => {
+          onTagsChange(this.getTagList());
+        },
+        onChipDelete: () => {
+          onTagsChange(this.getTagList());
+        },
+      },
+    );
+  }
+
+  getTagList = () => {
+    const tags = [];
+    this.instance.chipsData.forEach((chip) => {
+      tags.push(chip.tag);
+    });
+    return tags;
+  };
+
+  render() {
+    const { visible, onPublish } = this.props;
+    return (
+      <div className="publish-dropdown" style={{ display: visible ? 'block' : 'none' }}>
```

Comment:
Let's try and avoid using inline styles. Create a class for the above styles instead

Reply:
Let this be the case for every instance an inline style has been used

Reply:
I  have added a fix for that @Arthur236